packages/resolve-scripts/src/scripts/dev.server.config.js

```diff
@@ -4,6 +4,18 @@ const StartServerPlugin = require('start-server-webpack-plugin');
 
 const webpackServerConfig = require('../configs/webpack.server.config');
 
+if (!webpackServerConfig.plugins) {
+    webpackServerConfig.plugins = [];
+}
+
+webpackServerConfig.plugins.push(
+    new webpack.BannerPlugin({
```

Comment:
Why is not devtools used? (Like here: https://webpack.js.org/guides/development/#using-source-maps)